src/test_utils.rs

```diff
@@ -838,6 +932,7 @@ fn print_event(event: &Event) {
     let context_names = CONTEXT_NAMES.read().unwrap();
     match context_names.get(&event.id) {
         Some(name) => println!("{} {}", name, msg),
+        None if event.id == SEC_EVENT_ID => println!("\n========== {} ==========", msg),
```

Comment:
Oh, this is really strange. You are comparing *Context ID* to 0 and add `=====` around if they are equal. Why not just add these decorations in the `sec()` function?

Reply:
I pushed a commit removing `SEC_EVENT_ID`, @Hocuri what do you think about it?

Reply:
The downside is that it puts the newline somewhere else. With SEC_EVENT_ID: 
```
bob Received Event { id: 396827516, typ: IncomingMsg { chat_id: ChatId(10), msg_id: MsgId(10) } }

========== INFO: bob sends a message 'Hi back' to alice ==========
bob Received CHAT_MODIFIED(Chat#10)
```
With the latest committ:
```
bob Received Event { id: 1865664388, typ: IncomingMsg { chat_id: ChatId(10), msg_id: MsgId(10) } }
INFO: 
========== bob sends a message 'Hi back' to alice ==========
bob Received CHAT_MODIFIED(Chat#10)
```

(excerpt from the test at https://github.com/deltachat/deltachat-core-rust/pull/3385/)

I.e. there is no completely blank new line anymore, which makes it stand out less.

Something else we could do to solve this is just always leave out the "INFO: " because Info is just a generic log message, and there is not that much value added by always putting "INFO: ".

Reply:
I added another commit introducing `Section` event type, thoughts?

Maybe log sink should accept a type `LogEvent` that is either a `Section` or `Event`, but that's too much for this PR.

Reply:
Great, thanks!

Reply:
Oh, but now rust-analyzer shows me 4 errors in `deltachat-ffi/lib.rs` ("missing match arm") because of the `match event`s. That's not actually a compiler error because we don't compile the FFI in cfg(test), but it would still be annoying to be shown 4 false-positives in r-a forever.

We could add a cfg(test) match arm there that just aborts or whatever, which would be kinda weird but solve the problem.

> Maybe log sink should accept a type LogEvent that is either a Section or Event

or this.

Reply:
Added `LogEvent`

Reply:
Nice, thanks!